tick/sym.q

```diff
@@ -0,0 +1,2 @@
+// trade table schema
+trade:([]time:`timespan$();sym:`g#`symbol$();bid:`float$();ask:`float$());
```

Comment:
I think you have added this file involuntarily

Reply:
True, thanks!